### PR TITLE
Added note about JDK 8+ requirement

### PR DESCRIPTION
**Pull request by bramp**

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -4,10 +4,14 @@ layout: documentation
 ---
 
 Our goal is to make it simple to add Error Prone checks to your existing Java
-compilation. Please join our [mailing
+compilation. However, please note Error Prone requires JDK 8 or newer. It can be used to build java 6 and 7 code by setting the appropriate `-source` / `-target` / `-bootclasspath` flags.
```
Comment (eaftan) [inline docs/installation.md]: Can you please rephrase this as:

```
Please note that Error Prone must be run on JDK 8 or newer.  It can be used to build Java 6 or 7 code ...
```

**Comment by bramp** [comment]
I've updated the wording based on your suggestions. Please take a look.
